includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -120,30 +119,16 @@ public function init_form_fields() {
 	}
 
 	/**
-	 * Load admin scripts.
-	 *
-	 * @since x.x.x
-	 * @version x.x.x
+	 * Maybe override the parent admin_options method.
 	 */
-	public function admin_scripts() {
```

Comment:
It looks like this was duplicated between the `WC_Stripe_UPE_Payment_Gateway` and the `WC_Gateway_Stripe` class - refactored and moved to the `WC_Stripe_Settings_Controller` class so that they're both using the same logic to enqueue the scripts.

Reply:
This makes sense and thanks for the refactor!